File: src/main/java/kr/co/bookand/backend/feedback/repository/FeedbackRepository.java
package kr.co.bookand.backend.feedback.repository;

import kr.co.bookand.backend.feedback.domain.Feedback;
import org.springframework.data.jpa.repository.JpaRepository;

public interface FeedbackRepository extends JpaRepository<Feedback, Long> {
    Long countAllByVisibility(boolean visibility);

    Long countAllByVisibilityAndCreatedAtBetween(boolean visibility, String startDatetime, String endDatetime);
}

File: src/main/kotlin/kr/co/bookand/backend/feedback/repository/FeedbackRepository.kt
package kr.co.bookand.backend.feedback.repository

import kr.co.bookand.backend.feedback.model.Feedback
import org.springframework.data.jpa.repository.JpaRepository
import java.time.LocalDateTime

interface FeedbackRepository : JpaRepository<Feedback, Long> {

    fun countAllByVisibility(visibility: Boolean): Long

    fun countAllByVisibilityAndCreatedAtBetween(
        visibility: Boolean,
        startDatetime: LocalDateTime,
        endDatetime: LocalDateTime
    ): Long
}